src/bp/common/validation.ts

```diff
@@ -97,3 +97,12 @@ export const WorkspaceCreationSchema = Joi.object().keys({
     .valid(Object.keys(defaultPipelines))
     .default('none')
 })
+
+export const PipelineSchema = Joi.array().items({
+  id: Joi.string().required(),
+  label: Joi.string().required(),
+  action: Joi.string().required(),
+  reviewers: Joi.array().items(Joi.string()),
+  minimumApprovals: Joi.number(),
+  reviewSequence: Joi.string()
```

Comment:
Since there are only two possible choices, it should also make sure it's either one of the possible choice